honeybee_grasshopper_core/src/HB Add Shade.py

```diff
@@ -0,0 +1,48 @@
+# Honeybee: A Plugin for Environmental Analysis (GPL)
+# This file is part of Honeybee.
+#
+# Copyright (c) 2019, Ladybug Tools.
+# You should have received a copy of the GNU General Public License
+# along with Honeybee; If not, see <http://www.gnu.org/licenses/>.
+# 
+# @license GPL-3.0+ <http://spdx.org/licenses/GPL-3.0+>
+
+"""
+Add a Honeybee Shades to an Aperture, Face or Room.
```

Comment:
I agree that it's been confusing and that's a major reason why I'm separating them into two completely different component categories in Honeybee[+] and two completely different libraries in Python (honeybee-core vs. honeybee-energy). It was really a mistake that I merged these into the same component in honeybee-legacy. Given this large separation, I think that bringing up honeybee-energy blinds here is actually going to be more confusing to new users than being helpful (it would be like bringing up BSDF materials here as another way to model Venetian blinds).

Maybe we can reassess if this is necessary after you see what I have planned on the energy tab but, for now, I would prefer to leave it out.

Reply:
I actually think it will be very helpful. Separation of components doesn't show up in a script. Let's hope your assumption will be correct.